org-node/src/main.rs

```diff
@@ -26,6 +26,10 @@ pub struct Options {
     #[argh(option)]
     pub subgraph: String,
 
+    /// JSON-RPC WebSocket URL of Ethereum node (eg. ws://localhost:8545)
+    #[argh(option)]
+    pub rpc_url: String,
+
     /// poll interval for subgraph updates (seconds)
     #[argh(option)]
     pub poll_interval: Option<u64>,
```

Comment:
I think we can safely remove the poll_interval since we are no longer polling the subgraph